extensions/data-plane/data-plane-validation-facade/data-plane-validation-server/src/main/java/org/eclipse/dataspaceconnector/dataplane/validation/server/DataPlaneValidationServerExtension.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *  Copyright (c) 2021 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Amadeus - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.dataplane.validation.server;
+
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.jwk.ECKey;
+import com.nimbusds.jose.jwk.JWK;
+import org.eclipse.dataspaceconnector.dataplane.validation.server.controller.DataPlaneValidationFacadeController;
+import org.eclipse.dataspaceconnector.dataplane.validation.server.rules.ContractValidationRule;
+import org.eclipse.dataspaceconnector.dataplane.validation.server.rules.ExpirationDateValidationRule;
+import org.eclipse.dataspaceconnector.spi.EdcException;
+import org.eclipse.dataspaceconnector.spi.EdcSetting;
+import org.eclipse.dataspaceconnector.spi.WebService;
+import org.eclipse.dataspaceconnector.spi.contract.negotiation.store.ContractNegotiationStore;
+import org.eclipse.dataspaceconnector.spi.security.Vault;
+import org.eclipse.dataspaceconnector.spi.system.Inject;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtension;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtensionContext;
+import org.eclipse.dataspaceconnector.token.validation.impl.TokenValidationServiceImpl;
+
+import java.security.PublicKey;
+import java.util.Arrays;
+
+/**
+ * Provides service to validate and decode tokens.
+ */
+public class DataPlaneValidationServerExtension implements ServiceExtension {
```

Comment:
The `@Provides` annotation is missing

Reply:
This extension only registers a new controller, thus it does not provide anything.

Reply:
yes, same as before, I was confused, my fault